master/src/main/scala/org/apache/celeborn/service/deploy/master/clustermeta/ha/MasterNode.scala

```diff
@@ -41,15 +41,9 @@ case class MasterNode(
 
   def rpcEndpoint: String = rpcHost + ":" + rpcPort
 
-  def rpcIpEndpoint: String = rpcAddr.getAddress.getHostAddress + ":" + rpcPort
-
   def internalRpcEndpoint: String = rpcHost + ":" + internalRpcPort
 
-  def internalRpcIpEndpoint: String = rpcAddr.getAddress.getHostAddress + ":" + rpcPort
-
   lazy val ratisAddr = MasterNode.createSocketAddr(ratisHost, ratisPort)
-
-  lazy val rpcAddr = MasterNode.createSocketAddr(rpcHost, rpcPort)
```

Comment:
These are not used, and I cleaned them up as well.

Reply:
@RexXiong this is added in https://github.com/apache/celeborn/commit/993d3f2587b1c9cd8acd1a593a186d8044487c14, seems was never used

Reply:
Thanks for cleaning unused code!